doc/gh-pages/pages/development/brainflow.md

```diff
@@ -15,7 +15,11 @@ This plugin adds BrainFlow data acquisition SDK to MNE-SCAN app.
 
 ## Compilation of BrainFlow submodule
 
-* Make sure that you have brainflow git submodule
+* Make sure that you have brainflow git submodule by typing
+
+    $ git submodule init
```

Comment:
it looks like you missed code block symbols here "```" and I have these commands below

Reply:
I fixed it, I hope you dont mind

Reply:
not at all. Thx